packages/react-instantsearch-dom/src/components/__tests__/Panel.js

```diff
@@ -70,37 +70,42 @@ describe('Panel', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('expect to expose context to his children', () => {
-    const wrapper = shallow(
+  it('expect to expose context to its children', () => {
+    let provided;
+    const wrapper = mount(
       <Panel>
-        <div>Hello content</div>
+        <PanelConsumer>
+          {setCanRefine => {
+            provided = setCanRefine;
+            return null;
+          }}
+        </PanelConsumer>
       </Panel>
     );
 
-    expect(wrapper.instance().getChildContext()).toEqual({
-      setCanRefine: wrapper.instance().setCanRefine,
-    });
+    expect(provided).toEqual(wrapper.instance().setCanRefine);
   });
 
   it('expect to render when setCanRefine is called', () => {
-    const wrapper = shallow(
+    const wrapper = mount(
       <Panel>
-        <div>Hello content</div>
+        <PanelConsumer>
+          {setCanRefine => (
+            <button onClick={() => setCanRefine(false)}>set to false</button>
```

Comment:
We might expect the value `false` to change over time but it doesn't. 

I would rather use a simpler value for the text.